src/leakybucket.h

```diff
@@ -0,0 +1,121 @@
+// Copyright (c) 2015 The Bitcoin XT developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_LEAKYBUCKET_H
+#define BITCOIN_LEAKYBUCKET_H
+
+#if defined(HAVE_CONFIG_H)
+#include "config/bitcoin-config.h"
+#endif
+
+/** Default value for the maximum amount of data that can be received in a burst */
+static const int64_t DEFAULT_MAX_RECV_BURST = 100*1024; //*1024;
```

Comment:
the //*1024 is confusing here, as the comments don't specify what the metric is. I'm assuming all these values are in bytes? Or does the comment imply they're measured in kilobytes?